### PR TITLE
Support getting ticket counts by day over a specified period

### PR DESCRIPTION
**Pull request by jnormington**

### CODE REVIEW

File: zendesk/reports.go [hunk 1]
```diff
@@ -152,6 +157,52 @@ func detailedMetrics(r *conf.Report, c *conf.Config) error {
...
+
+	tp, err := client.SearchTickets(&Query{Params: r.Filter.BuildQuery(&timeNow)})
+	if err != nil {
+		return err
+	}
+
+	var found bool
+	for _, t := range tp.Tickets {
+		td := t.CreatedAt.Format(dateFormat)
+
```
Comment (achilleasa) [inline zendesk/reports.go]: Maybe this would look better like this (its a 🏄  though...)

found := false

File: zendesk/reports.go [hunk 2]
```diff
@@ -152,6 +157,52 @@ func detailedMetrics(r *conf.Report, c *conf.Config) error {
...
+		for i, dc := range gbData {
+			if td == dc.Date {
+				gbData[i].Count++
+				found = true
+				break
+			}
+		}
+
+		if !found {
```
Comment (achilleasa) [inline zendesk/reports.go]:
```
if found { 
   continue
}

gbData = append(gbData, DateData{Date: td, Count: 1})

File: zendesk/reports.go [hunk 3]
```diff
@@ -152,6 +157,52 @@ func detailedMetrics(r *conf.Report, c *conf.Config) error {
...
+			gbData = append(gbData, DateData{Date: td, Count: 1})
+		}
+
+		found = false
```
Comment (achilleasa) [inline zendesk/reports.go]: this can be removed (so does the `var found` line outside of the loop)

**Comment by achilleasa** [comment]
Left a small 🏄  but otherwise LGTM

**Comment by jnormington** [comment]
Thanks Achilleas for the comments updated.

**Comment by jnormington** [comment]
Closing PR to repoint to master...

**Pull request closed without merging**